src/azul/indexer/aggregate.py

```diff
@@ -159,7 +159,13 @@ class SetOfDictAccumulator(SetAccumulator):
     """
 
     def accumulate(self, value) -> bool:
-        return super().accumulate(freeze(value))
+        frozen_value = freeze(value)
+        # tuples are not recognized as collections by the superclass, so without
```

Comment:
This seem convoluted. The comment mentions tuples while the conditional looks for lists and examines the argument rather than the return value. Additionally, the comment immediately raises the question as to why we can't make the superclass handle those tuples.

I think we should start adding doctests to the various accumulators. In this PR we only need to cover the accumulator classes that are being modified. One of the doctests should cover the scenario that is being fixed.

Reply:
Decided at PL to try an alternative approach wherein `sequence_cls` is added as a parameter to `json_freeze.freeze` to avoid the intermediary tuple representation of the frozen value.

If that doesn't work, current approach is acceptable, but comments need to added/improved to address @hannes-ucsc's question in the above comment.